src/Service.Host/client/src/components/salesArticles/SalesArticles.js

```diff
@@ -0,0 +1,45 @@
+﻿import React, { Component } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { withStyles } from '@material-ui/core/styles';
+import Typeahead from './Typeahead';
+import Paper from '@material-ui/core/Paper';
+
+const styles = theme => ({
+    paper: {
+        padding: theme.spacing.unit * 6
+    },
+    boldHeader: {
```

Comment:
are all these styles still used now the typeahead has moved into it's own component?  I can see classes.paper still referenced but not sure about the rest

Reply:
good point. I'll check it out